timeseries/src/autogluon/timeseries/models/ensemble/array_based/regressor/linear_stacker.py

```diff
@@ -42,16 +46,18 @@ def __init__(
         lr: float = 0.1,
         max_epochs: int = 10_000,
         relative_tolerance: float = 1e-7,
+        prune_below: float = 0.0,
     ):
         super().__init__()
         self.quantile_levels = quantile_levels
         self.weights_per = weights_per
         self.lr = lr
         self.max_epochs = max_epochs
         self.relative_tolerance = relative_tolerance
+        self.prune_below = prune_below
 
-        # Learned weights (stored as numpy arrays)
         self.weights: np.ndarray | None = None
+        self.kept_indices: list[int] | None = None
```

Comment:
Should we make this an attribute of the abstract class `EnsembleRegressor` for future-proofing? We could use similar logic in e.g. a GBDT-based TabularRegressor by looking at `model.feature_importances_`.

Reply:
will skip for now as EnsembleRegressor has a dummy constructor.